testing/xfpga/test_sysfs_c.cpp

```diff
@@ -748,8 +748,66 @@ TEST_P(sysfs_c_mock_p, fpga_sysfs_03) {
   EXPECT_EQ(result, FPGA_OK);
 }
 
+TEST_P(sysfs_c_mock_p, fpga_sysfs_04) {
```

Comment:
Can we add some test descriptions to these tests?

Reply:
fixed